messenger.go

```diff
@@ -319,17 +341,27 @@ func (m *TGMessenger) ProcessCommand(message *tgbotapi.Message, chat *Chat) {
 	}
 }
 
-func (m *VKMessenger) processWall(wall object.WallWallpost, chatID int64, message *Message) {
-	if message.Text != "" {
-		message.Text += "\n"
+func (m *VKMessenger) getWallAuthor(wall *object.WallWallpost) string {
+	userResponce, err := m.vk.GroupsGetByID(api.Params{"user_ids": wall.FromID})
+	if err != nil {
+		log.Print("could not find community with id ", wall.FromID)
+		return ""
+	}
+	return concatenateMessageSender(userResponce[0].Name, "vk")
+}
+
+func (m *VKMessenger) processWall(wall object.WallWallpost, chat *Chat) {
+	message := Message{
+		Text:   wall.Text,
+		Sender: m.getWallAuthor(&wall),
```

Comment:
i believe we should just use public name instead of chat name. Looks like getting rid of sender name is a bad idea

Reply:
Sender's name is included in another message, which contains their text, and is normally sent right before the "wall message"